omod/src/main/webapp/resources/js/ugandaemr-poc.js

```diff
@@ -202,4 +202,140 @@ function enable_disable(field, class_name_prefix, conditions, input_type) {
             }
         );
     });
+}
+
+
+
+function enable_disable_district_tb_reg(selected_option) {
```

Comment:
@semujju Wat is the purpose of this function?

Reply:
Its one of those functions you asked me to extract to the global js file. Its purpose is to enable fields based on TB RX